api_volontaria/local_settings.py

```diff
@@ -0,0 +1,4 @@
+# Local_settings.py
+# You can use this file to override the global settings variable that are not taking care inside the .env file
+# Follow this link for more informations:
+# https://github.com/Volontaria/API-Volontaria/blob/master/docs/getting_started/virtualenv.md#custom-settings
```

Comment:
Link should use the readable documentation hosted on Github to be more easy to use: https://volontaria.github.io/API-Volontaria/getting_started/virtualenv/#custom-settings